src/api.ts

```diff
@@ -498,6 +498,8 @@ export type CreateUserRequest = {
     username?: string,
     firstName?: string,
     lastName?: string,
+
+    properties?: { [key: string]: any },
```

Comment:
Can we use `string | number | boolean` instead of any? or create a type somewhere that encapsulates those three (`userPropertyValue` or something)

Reply:
As far as I know, there's not actually a JSON type. We can enumerate the possibilities, but I don't think it gets you that much since you still would need to cast the values to use it.